contract/contracts/PixelDevsUkraineDonation.sol

```diff
@@ -35,8 +36,10 @@ contract PixelDevsUkraineDonation is
         uint256 indexed newValue
     );
     event ContractStateUpdated(bool indexed oldValue, bool indexed newValue);
+    event WithdrawWalletUpdated(address indexed oldValue, address indexed newValue);
```

Comment:
This can be removed too to be fair when we are hard coding the Ukraine DAO treasury address